spotify_player/src/command.rs

```diff
@@ -5,6 +5,7 @@ use crate::state::{
 use serde::Deserialize;
 
 #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+#[serde(tag = "type", content = "args")]
```

Comment:
let's avoid making any breaking changes? Why do we need tagging? By default, the syntax would be

```
[[keymaps]]
key_sequence = "+"
command = { VolumeChange: { offset = 1 } }
```

while other commands are the same

Reply:
Ahhhh, thanks. I couldn't figure out the syntax for it.

Although you do seem to have a typo there, since the `:` needs to be a `=`

Reply:
I will make the changes to the docs and then push the changes to be merged.

Reply:
> Although you do seem to have a typo there, since the `:` needs to be a `=`

Yep, should be `=`